README.md

```diff
@@ -159,6 +159,25 @@ This pipeline will:
    with an empty file.
  * Destroy the resources created by terraform.
 
+#### Deploy cloudfoundry with microbosh
+```
+# Optionally pass the current branch for the git resources
+export BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
There is a section below talking about setting the BRANCH variable. Shall we refer to it here?